streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamTask.java

```diff
@@ -621,6 +622,7 @@ void suspend(final boolean clean,
             try {
                 commit(false, partitionTimes);
             } finally {
+                partitionGroup.clear();
```

Comment:
Previously, we clear the `partitionGroup` within `closeTopology()` that we call above -- however, because of the consumer position tracking, we need to delay it after the commit.